frontend/src/pages/GuidePage.tsx

```diff
@@ -0,0 +1,35 @@
+import Tabs from '@/components/tab/Tabs';
+import HowToFork from '@/components/guide/HowToFork';
+import { TabData } from '@/types';
+
+//TODO 여기서만 쓰일 데이터 같아서 일단 위에 선언해놓습니다. @버건디
+
+const TAB_LIST: TabData[] = [
```

Comment:
Guide와 관련된 Tab List라는 걸 드러내기 위해 GUIDE라는 단어를 앞에 붙여주는 건 어떻게 생각하시나요~?
```suggestion
const GUIDE_TAB_LIST: TabData[] = [
```

Reply:
TabInfo 라고 변경해보았습니다. 짚어주셔서 감사합니다~! 🙇